pkg/syncer/syncer.go

```diff
@@ -190,11 +192,11 @@ func StartSyncer(ctx context.Context, cfg *SyncerConfig, numSyncerThreads int, i
 	// Check whether we're in the Advanced Scheduling feature-gated mode.
 	advancedSchedulingEnabled := false
 	if syncTarget.GetAnnotations()[AdvancedSchedulingFeatureAnnotation] == "true" {
-		klog.Infof("Advanced Scheduling feature is enabled for syncTarget %s", cfg.SyncTargetName)
+		logger.Info("Advanced Scheduling feature is enabled")
 		advancedSchedulingEnabled = true
 	}
 
-	klog.Infof("Creating spec syncer for SyncTarget %s|%s, resources %v", cfg.SyncTargetWorkspace, cfg.SyncTargetName, resources)
+	logger.Info(fmt.Sprintf("Creating spec syncer resources %v", resources))
```

Comment:
nit: as a rule of thumb, `fmt.Sprintf()` in structured logs is a smell - attach the data you're formatting as fields